presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -435,10 +435,37 @@ Create an Iceberg table partitioned by the first letter of the team field::
         partitioning = ARRAY['truncate(team, 1)']
     );
 
+Create an Iceberg table partitioned by ``ds``::
+
+    CREATE TABLE players (
+        id int,
+        name varchar,
+        team varchar,
+        ds date
+    )
+    WITH (
+        format = 'ORC',
+        partitioning = ARRAY['year(ds)']
+    );
+
+Create an Iceberg table partitioned by ``ts``::
+
+    CREATE TABLE players (
+        id int,
+        name varchar,
+        team varchar,
+        ts timestamp
+    )
+    WITH (
+        format = 'ORC',
+        partitioning = ARRAY['hour(ts)']
+    );
 .. note::
 
-    ``Day``, ``Month``, ``Year``, ``Hour`` partition column transform functions are not supported in Presto Iceberg
-    connector yet (:issue:`20570`).
+    ``Day``, ``Month``, ``Year``, ``Hour`` partition column transform functions are supported for timestamp type in Presto Iceberg
```

Comment:
Let's remove these lines.

Reply:
Please add the documentation below line 412.  It seems `identity` is missing there as well, please feel free to add that!